src/data/types/toastMessage.types.ts

```diff
@@ -1,8 +1,5 @@
 export enum TOAST_MESSAGE {
```

Comment:
```suggestion
 export enum PRODUCT_TOAST_MESSAGE {
```
Все таки под кастомеров и ордера свои тосты